web/src/backend/FetchFilter.js

```diff
@@ -21,35 +21,38 @@ import * as Setting from "../Setting";
 const {confirm} = Modal;
 const {fetch: originalFetch} = window;
 
-const demoModeCallback = async(url, option) => {
-  if (option.method === "POST") {
-    confirm({
-      title: i18next.t("general:This is a read-only demo site!"),
-      icon: <ExclamationCircleFilled />,
-      content: i18next.t("general:Go to writable demo site?"),
-      okText: i18next.t("user:OK"),
-      cancelText: i18next.t("general:Cancel"),
-      onOk() {
-        Setting.openLink(`https://demo.casdoor.com${location.path}${location.search}?username=built-in/admin&password=123`);
-      },
-      onCancel() {},
-    });
-  }
-
-  return option;
+const demoModeCallback = (res) => {
+  res.json().then(data => {
+    if (data.msg === "Unauthorized operation") {
```

Comment:
This string is i18n, not only English:

![image](https://user-images.githubusercontent.com/3787410/219828690-fa4ecd4f-506c-43e0-afa5-0ad2af352ff0.png)